proposals/drafts/elections.rst

```diff
@@ -1,37 +1,249 @@
-This proposes to add some election related objects to OCD. It attempts to stay close to the VIP election specification. http://vip-specification.readthedocs.io/en/vip5/xml/index.html#elements
+====================
+OCDEP: Elections
+====================
+
+:Created: 2016-12-28
+:Author: James Gordon, Forest Gregg, `California Civic Data Coalition <http://www.californiacivicdata.org/>`_
+:Status: Proposed
+
+Overview
+========
+
+Definition of data types to model elections within which political contests are decided, including relationships to ballot measures and candidates competing for public office.
+
+The proposed data types are:
+
+* ``Election``
+* ``ContestBase``, a base class for:
+
+    - ``BallotMeasureContest``
+    - ``CandidateContest``
+    - ``PartyContest``
+    - ``RetentionContest``
+
+* ``Candidacy``
+* ``Party``
+* ``BallotSelectionBase``, a base class for:
+
+    - ``BallotMeasureSelection``
+    - ``CandidateSelection``
+    - ``PartySelection``
```

Comment:
It's the same in VIP. This is covered by `BallotMeasureSelection` since `RententionContest` subclasses `BallotMeasureContest`.